test/test_helper.rb

```diff
@@ -1,4 +1,5 @@
 $LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
+require 'circuitbox'
```

Comment:
just curious, why was this necessary?

Reply:
Because in the tax area test we only use the circuit if `circuitbox` is defined, which it is not when the test is run individually. The fact that the suite passes without this line currently is because of ordering. Will add a few more fixes as part of this PR too since the change is minor.